lib/oli/publishing.ex

```diff
@@ -1489,7 +1489,8 @@ defmodule Oli.Publishing do
   Returns a map of activity_ids to a map containing the slug and resource id of the
   page that encloses it
   """
-  def determine_parent_pages(activity_resource_ids, publication_id) do
+  def determine_parent_pages(activity_resource_ids, publication_ids)
```

Comment:
List support is added here, but it doesn't seem to be used anywhere.  If it isn't being used, we should revert these changes.

Reply:
That is used in the `defp determine_parent_link(section_slug, activity_id) do` at `defmodule Oli.Rendering.Report.Html do`